testSuite/scripts/utility.py

```diff
@@ -25,6 +25,16 @@ def add_arguments(self, argument):
         if argument == None:
             return
         self.args.append(argument)
+
+        # auto-set MD5 checking flags, so that we always check when testing
+        ct = str.lower(self.command_type)
+        is_copy_or_sync = ct == "copy" or ct == "cp" or ct == "sync"
```

Comment:
>is_copy_or_sync = ct == "copy" or ct == "cp" or ct == "sync" [](start = 8, length = 60)

I don't believe that md5 checks occur on S2S transfers, at least from S3 to Azure.